### PR TITLE
Add scheduled job to handle rejected files

### PR DESCRIPTION
**Pull request by rkondratowicz**
https://tools.hmcts.net/jira/browse/BPS-1003

chart changes: https://github.com/hmcts/blob-router-service/pull/191
flux changes: hmcts/cnp-flux-config#1966

### CODE REVIEW

File: charts/reform-scan-blob-router/values.yaml [hunk 1]
```diff
@@ -28,6 +28,7 @@ java:
     FLYWAY_SKIP_MIGRATIONS: true
 
     DELETE_DISPATCHED_FILES_CRON: "0 0/10 * * * *"
+    HANDLE_REJECTED_FILES_CRON: "0 0 7 * * *"
```
Comment (doncem) [inline charts/reform-scan-blob-router/values.yaml]: Changes to `values.yaml` requires chart version bump + separate PR + flux PR

**Comment by nitinprabhu** [comment]
> chart changes better go with separate PR,
> bump the version,
> do we need to override new var in flux for AAT and/or prod?

good point. chart changes it would be nice if chart changes goes in separate PR plus flux changes. Then we can update tests/code.

**Comment by rkondratowicz** [comment]
Flux PR: https://github.com/hmcts/cnp-flux-config/pull/1966

**Comment by doncem** [comment]
> Flux PR: [hmcts/cnp-flux-config#1966](https://github.com/hmcts/cnp-flux-config/pull/1966)

Still config related changes should be as separate PR 😉

**Comment by rkondratowicz** [comment]
> Still config related changes should be as separate PR 😉

ok, on it

**Comment by rkondratowicz** [comment]
chart config changes:
https://github.com/hmcts/blob-router-service/pull/188

**Comment by rkondratowicz** [comment]
All ready now

**Pull request merged**